### PR TITLE
Initialize components

### PR DESCRIPTION
**Pull request by SimonGrchevski**
- Add the header 
- Add the library content
- Add the categories content
- Add the basic styles
- Add the logic for adding a new book
- Fix linter errors

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      If you open it directly in the browser, you will see an empty page.
+
+      You can add webfonts, meta tags, or analytics to this file.
+      The build step will place the bundled scripts into the <body> tag.
+
+      To begin the development, run `npm start` or `yarn start`.
+      To create a production bundle, use `npm run build` or `yarn build`.
+    -->
+  </body>
+</html>
```
Comment (Emmyn5600) [inline public/index.html]: - [x] Remove unnecessary files or assets in the public folder.
react20

File: public/manifest.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+      "sizes": "192x192"
+    },
+    {
+      "src": "logo512.png",
+      "type": "image/png",
+      "sizes": "512x512"
+    }
+  ],
+  "start_url": ".",
+  "display": "standalone",
```
Comment (Emmyn5600) [inline public/manifest.json]: - [x] Kindly, let us restructure or readjust again our root folders as suggested.

-------------------------------------------------------Yours------------------------------------------------
react20

-------------------------------------------------------Original------------------------------------------------
react21

File: src/components/bookInfo.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+      </div>
+      <div className="optionsSection">
+        <ul className="flex optionsUl">
+          <li>Comments</li>
+          <li>Remove</li>
+          <li>Edit</li>
+        </ul>
+      </div>
+    </article>
+  );
```
Comment (Emmyn5600) [inline src/components/bookInfo.js]: - [x] Book Has a button for removing books, the button wouldn't remove the books yet, this is just a placeholder for now nice however its a button not a text, make it as pointer.
react22

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED :recycle:

Hi, @SimonGrchevski 

Great work on implementing the project requirements, however there are some issues to be solved before you get approved but you are almost there:muscle:. 
   
   - The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

Changes required :writing_hand: 
 
      * #### Please check the comments under the review.
       
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @SimonGrchevski 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**